app/connectors/CustomsDeclareExportsMovementsConnector.scala

```diff
@@ -65,22 +64,22 @@ class CustomsDeclareExportsMovementsConnector @Inject()(appConfig: AppConfig, ht
   )(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[HttpResponse] =
     postRequest(DucrDisassociation, requestXml)
 
-  def sendShutMucrRequest(requestXml: String)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[HttpResponse] =
-    postRequest(ShutMucr, requestXml)
-
   private def postRequest(
     actionType: ActionType,
-    requestXml: String
+    payload: String
```

Comment:
from my perspective we should have her as much detailed type as we can - here we would be able to POST "fizzbuzz" and type system would not prevent before it

Reply:
Why are we using `PostString`? its not xml any more so we can post JSON

Reply:
:man_facepalming: Good point.